compose/project.py

```diff
@@ -54,7 +54,7 @@ def visit(n):
     return sorted_services
 
 
-class Project(object):
+class Project(ParallelContainerManager):
```

Comment:
I would love it if we could avoid inheritance here. I think an instance of`ParallelContainerManager()` could be an attribute instead of a base class. The only method it uses from `Project` is `self.containers()`. Instead of passing in the service of service_names to the calls, the list of containers could be passed in.

`Project.stop()` could then call `self.parallel_manager.execute('stop', containers, **options)`